source/component/PasDoc_Items.pas

```diff
@@ -2859,11 +2859,24 @@ procedure TPasRoutine.Serialize(const ADestination: TStream);
 end;
 
 function TPasRoutine.Signature: string;
+var
+  I: Integer;
 begin
+  ParamTypes.QuoteChar := #0;
+  ParamTypes.Delimiter := ',';
+
   if ParamTypes.Count > 0 then
-    Result := Name + '(' + ParamTypes.CommaText + ')'
+    Result := Name + '(' + ParamTypes.DelimitedText + ')'
   else
     Result := Name;
+
+  I := 1;
```

Comment:
There is already a very similar code in `NormalizeSignature` in `TPasRoutines.FindListItem`, that removes spaces from string.

Let's make a routine in `PasDoc_Utils` that can be used for this, like `function SRemoveChars(const S: String; const C: Char): String;`